modules/hitlnext/src/views/full/app/components/HandoffItem.tsx

```diff
@@ -16,7 +16,7 @@ import UserName from './UserName'
 
 const HandoffItem: FC<IHandoff> = ({ createdAt, id, status, agentId, userConversation, userChannel, user }) => {
   const { state, dispatch } = useContext(Context)
-
+  moment.locale(lang.getLocale())
```

Comment:
what's this for ?

Reply:
This is to instruct moment to display elapsed time in the selected language. In spanish it is "dias", "horas", "segundos"
I think the same apply to other languages.